### PR TITLE
Fix: Document platform requirements

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] documents platform requirements

### CODE REVIEW

**Comment by mcop1** [comment]
@localheinz Thank you for your contribution, could you please merge the 1.x branch into your pr again? This should fix the failing tests.

File: composer.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     }
   },
   "require": {
+    "php": "~7.4.0 || ~8.1.0 || ~8.2.0",
```
Comment (blankse) [inline composer.json]:
```suggestion
    "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
```

File: composer.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
     }
   },
   "require": {
+    "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
```
Comment (localheinz) [inline composer.json]: Does this project support PHP 8.2 yet?

Running

```shell
git grep 8.2
```

on current `1.x` yields nothing.

Comment (mcop1) [reply composer.json]: Not yet, we are currently working on it for the core https://github.com/pimcore/pimcore/pull/14080

**Comment by mcop1** [comment]
Thanks 👍

**Pull request merged**